setup.py

```diff
@@ -81,17 +72,19 @@ def do_run(self, builder: str, output_dir: str):
         rmtree(correct_doctrees, ignore_errors=True)
         move(self.doctree_dir, correct_doctrees)
 
+
 cmdclass = {}
 cmdclass["build_sphinx"] = MakeDocs
 
 
 install_requires = [
     'sphinx==2.0.*',
-     'cylc==8.0.*',
+    'cylc-flow==8.0a0',
 ]
 
 
 setup(
+    name='Cylc Documentation',
```

Comment:
I am testing the PR from @hjoliver, and had to pull your changes too. Installing it works, and says "Successfully install cylc-doc" (surprised it didn't use "Cylc Documentation", and also didn't complain about the name having spaces), but `pip uninstall cylc-doc` complains about not finding a file with that name. `pip uninstall "Cylc Documentation".

The only way to uninstall it is via `pip uninstall cylc-documentation`. For me, that's a bug in pip/setuptools.. as it displays installed `cylc-doc` :confused: 

I think it might be simpler if the name was `cylc-doc`, or `cylc-documentation` perhaps?

Reply:
Dammit, I thought `name` was supposed to be a long name for text presentation.